app/api/helpers/permissions.py

```diff
@@ -87,5 +87,22 @@ def decorated_function(*args, **kwargs):
 
     return decorated_function
 
+@second_order_decorator(jwt_required)
+def is_organizer(f):
+    """
+    Allows only Organizer to access the event resources.
+    :param f:
+    :return:
+    """
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
 
+        if user.is_staff:
+            return f(*args, **kwargs)
+        if 'event_id' in kwargs and user.is_organizer(kwargs['event_id']):
+            return f(*args, **kwargs)
+        return ForbiddenError({'source': ''}, 'Organizer access is required').respond()
+
+    return decorated_function
```

Comment:
So should I write jwt_required for all - list, detail and relationship? @shubham-padia